libnopegl/src/nopegl.h.in

```diff
@@ -118,11 +119,26 @@ struct ngl_scene {
 NGL_API struct ngl_scene *ngl_scene_create(void);
 
 /**
- * Initialize the scene graph passing down the graph directly.
+ * Initialize the scene from parameters.
+ *
+ * Only the root node in the parameters is mandatory.
+ *
+ * Default parameters:
+ * - duration: 30 seconds
+ * - framerate: 60/1 FPS
+ * - aspect ratio: 1:1
+ *
+ * The reference counter of the root node will be incremented for the scene
+ * lifetime, or until another scene replaces it.
+ */
+NGL_API int ngl_scene_init_from_params(struct ngl_scene *s, const struct ngl_scene_params *params);
```

Comment:
What about calling this method simply `ngl_scene_init()` and rename `ngl_scene_init_from_str()`  `ngl_scene_deserialize()` so it is the counterpart of `ngl_scene_serialize()` ?

Reply:
I renamed `ngl_scene_init_from_params` to `ngl_scene_init()`, but I don't like `ngl_scene_deserialize()` because it hides the fact that it behaves exactly the same as `ngl_scene_init()`, it's just a different input.

Reply:
Ok. Thanks.